src/Surfnet/StepupSelfService/SelfServiceBundle/Service/YubikeySecondFactorService.php

```diff
@@ -67,11 +68,12 @@ public function provePossession(VerifyYubikeyOtpCommand $command)
         }
 
         $secondFactorId = Uuid::generate();
+        $yubikeyPublicId = YubikeyPublicId::fromModHex(substr($command->otp, 0, -32))->getYubikeyPublicId();
```

Comment:
again multiline please, and document as to why you take everything but the last 32 chars :wink: